ABRSchema_0-2-0.json

```diff
@@ -634,6 +678,76 @@
+            },
+            "Volumes": {
+                "$comment": "Update Plates whenever you add a new plate type. Inputs should have a `inputType` and `parameterName`.",
+                "type": "object",
+                "properties": {
+                    "Key Data": {
+                        "description": "Volume data source.",
+                        "properties": {
+                                "inputType": { "const": "IVLab.ABREngine.VolumeKeyData" },
+                                "parameterName": { "const": "Key Data" },
+                                "inputGenre": { "const": "KeyData" },
+                                "inputValue": { "$ref": "#/definitions/InputStringTypes/DataPath" }
+                        }
+                    },
+                    "Color Variable": {
+                        "description": "Scalar values that make up the volume.",
+                        "properties": {
+                                "inputType": { "const": "IVLab.ABREngine.ScalarDataVariable" },
+                                "parameterName": { "const": "Color" },
+                                "inputGenre": { "const": "Variable" },
+                                "inputValue": { "$ref": "#/definitions/InputStringTypes/DataPath" }
+                        }
+                    },
+                    "Colormap": {
+                        "description": "Defines the color of the volume at control points along its scalar range.",
+                        "properties": {
+                                "inputType": { "const": "IVLab.ABREngine.ColormapVisAsset" },
+                                "parameterName": { "const": "Color" },
+                                "inputGenre": { "const": "VisAsset" },
+                                "inputValue": { "$ref": "#/definitions/InputStringTypes/UUID" }
+                        }
+                    },
+                    "Opacitymap": {
+                        "description": "Defines the opacity of the volume at control points along its scalar range.",
+                        "properties": {
+                            "inputType": { "const": "IVLab.ABREngine.PrimitiveGradient" },
+                            "parameterName": { "const": "Color" },
+                            "inputGenre": { "const": "PrimitiveGradient" },
+                            "inputValue": { "$ref": "#/definitions/InputStringTypes/UUID" }
+                        }
+                    },
+                    "Volume Brightness": {
+                        "description": "Manual brightness adjustment for the volume",
+                        "properties": {
+                                "inputType": { "const": "IVLab.ABREngine.PercentPrimitive" },
+                                "parameterName": { "const": "Volume" },
+                                "inputValue": { "default": "10%", "$ref": "#/definitions/InputStringTypes/PercentPrimitive" },
```

Comment:
Having now tested this, the default here should probably be something like 50% instead of 10%?